x-pack/plugins/cases/common/constants.ts

```diff
@@ -7,16 +7,27 @@
 import { ConnectorTypes } from './api';
 import { CasesContextFeatures } from './ui/types';
 
-export const DEFAULT_DATE_FORMAT = 'dateFormat';
-export const DEFAULT_DATE_FORMAT_TZ = 'dateFormat:tz';
+export const DEFAULT_DATE_FORMAT = 'dateFormat' as const;
+export const DEFAULT_DATE_FORMAT_TZ = 'dateFormat:tz' as const;
 
-export const APP_ID = 'cases';
+/**
+ * Application
+ */
+
+export const APP_ID = 'cases' as const;
+export const APP_OWNER = 'cases' as const;
+export const APP_PATH = '/app/management/insightsAndAlerting/cases' as const;
+export const APP_PARENT_ID = 'management' as const;
```

Comment:
I am not sure what this `APP_PARENT_ID` is suppose to mean. Could you explain? maybe add a comment here too.

Reply:
Because cases in the stack management, it is a child of the management plugin. We register cases to the management plugin. This variable holds the application ID of the management plugin. I will put a comment.